### PR TITLE
Enable Clang-Format on CI for ungrouped apps

### PR DESCRIPTION
**Pull request by SunBlack**
Missed it in #3885 (as far as I remember I didn't added it there as there were many Clang-Format MRs to avoid merge conflicts)

### CODE REVIEW

File: apps/src/openni_mls_smoothing.cpp [hunk 1]
```diff
@@ -37,12 +37,12 @@
 #include <pcl/console/parse.h>
 #include <pcl/io/openni_camera/openni_driver.h>
 #include <pcl/io/openni_grabber.h>
+#include <pcl/search/kdtree.h> // for KdTree
 #include <pcl/surface/mls.h>
```
Comment (larshg) [inline apps/src/openni_mls_smoothing.cpp:41]: Any reason comments aren't aligned here, when they are in apps/include/pcl/apps/impl/dominant_plane_segmentation.hpp?

**Comment by larshg** [comment]
Apart from the seemingly inconsistency with "for" comments, LGTM

File: apps/src/openni_mls_smoothing.cpp (hunk 1 above)
Comment (SunBlack) [reply apps/src/openni_mls_smoothing.cpp:41]: I believe Clang-Format does this only, when there are comments in successive lines

**Review by mvieth — APPROVED** [review]

**Review by larshg — APPROVED** [review]

File: .dev/format.sh [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 format() {
     # don't use a directory with whitespace
-    local whitelist="apps/3d_rec_framework apps/modeler 2d geometry ml octree simulation stereo tracking registration gpu/containers"
+    local whitelist="apps/3d_rec_framework apps/include apps/modeler apps/src 2d geometry ml octree simulation stereo tracking registration gpu/containers"
```
Comment (kunaltyagi) [inline .dev/format.sh:11]: Can we use just apps now?

**Review by kunaltyagi — COMMENTED** [review]
LGTM otherwise

File: .dev/format.sh (hunk 2 above)
Comment (SunBlack) [reply .dev/format.sh:11]: Nop, cloud_composer, in_hand_scanner & modeler are still missing

**Pull request merged**